website/docs/r/cluster.html.markdown

```diff
@@ -206,6 +206,22 @@ Note free tier (M0) creation is not supported by the Atlas API and hence not sup
 * `auto_scaling_disk_gb_enabled` - (Optional) Specifies whether disk auto-scaling is enabled. The default is true.
     - Set to `true` to enable disk auto-scaling.
     - Set to `false` to disable disk auto-scaling.
+* `auto_scaling_compute_enabled` - (Optional) Specifies whether cluster tier auto-scaling is enabled. The default is false.
+    - Set to `true` to enable cluster tier auto-scaling. If enabled, you must specify a value for `providerSettings.autoScaling.compute.maxInstanceSize`.
+    - Set to `false` to disable cluster tier auto-scaling.
+    
+    -> **IMPORTANT:** If `auto_scaling_compute_enabled` is true,  then Atlas will automatically scale up to the maximum provided and down to the minimum, if provided.
+        This will cause the value of `provider_instance_size_name` returned to potential be different than what is specified in the Terraform config and if one then applies a plan, not noting this,
+         Terraform will scale the cluster back down to the original instanceSizeName value. 
+         To prevent this a lifecycle customization should be used, i.e.:
+        ```hcl
+          lifecycle {
+              ignore_changes = [provider_instance_size_name]
+          }  
```

Comment:
Just a few more adjustments - move the important block above "* `auto_scaling_compute_enabled`" - line 209.

-> **IMPORTANT:** If `auto_scaling_compute_enabled` is true,  then Atlas will automatically scale up, if needed, to the maximum provided and down to the minimum if `auto_scaling_compute_scale_down_enabled` is also true. This will cause the value of `provider_instance_size_name` returned to potentially be different than what is specified in the Terraform config and if one then applies a plan, not noticing this, Terraform will scale the cluster back down to the original `provider_instance_size_name` value.   To prevent this a lifecycle customization should be used, i.e.:
        ```hcl
          lifecycle {
              ignore_changes = [provider_instance_size_name]
          }

Reply:
done, thank you melissa

Reply:
Ty!!